addons/website_slides_survey/controllers/slides.py

```diff
@@ -3,52 +3,33 @@
 
 from odoo.addons.website_slides.controllers.main import WebsiteSlides
 
+from odoo import http
 from odoo.http import request
 
+import werkzeug.urls
+import werkzeug.utils
+from werkzeug.exceptions import BadRequest
+
 
 class WebsiteSlides(WebsiteSlides):
 
+    @http.route(['/slides_survey/certification_url/get'], type='http', auth='user', website=True)
```

Comment:
Maybe use `/slides_survey/slide/get_certification_url` to be consistent with other routes?
And call the method accordingly (`slide_get_certification_url`)

Reply:
Done